locustempus/templates/main/course_detail.html

```diff
@@ -36,14 +36,14 @@
             </div>
         </div>
         {% endif %}
-        <form method="post">
+        <form method="post" class="lt-layout-toggle">
             {% csrf_token %}
-            <button type="submit" class="btn lt-button mr-1 lt-button--transparent" {% if project_list_grid %}disabled{% endif %}>
-                <span class='lt-icons lt-button__icon'><img src="{{STATIC_URL}}img/icon-grid.svg" class="{% if project_list_grid %}inactive{% else %}active{% endif %}" /></span>
+            <button type="submit" class="lt-button mr-1 lt-button--transparent {% if project_grid_layout %}active{% endif %}" {% if project_grid_layout %}disabled{% endif %}>
```

Comment:
Let's have a quick convo about the active/disable state. To me, I think it makes sense to have the current state disabled, and the other state active so that it signals to users to click on it.